src/main/cpp/Robot.cpp

```diff
@@ -28,6 +28,8 @@ void Robot::RobotInit() {
 	m_DriveTrain.Init();
     m_oi.Init();
 
+	m_oi.m_FodToggle.WhenPressed(frc2::InstantCommand([&] { m_oi.FlipFOD(); }, {} ));
```

Comment:
Can we inline the flip boolean and the debug out into the lambda instead of writing a whole new method?